### ISSUE TITLE
Quoted arguments get splitted

### ISSUE THREAD

**Issue by martin-braun** [gonuit/rps#21]
A script of mine launches a dart program:

```yaml
scripts:
  screenshots: fvm dart flutter_screenshots.dart
```

Now I wish to call it like this:

```rps screenshots "iPhone Xs Max"```

However, the args are messed up:

```dart
Future<void> main(List<String> args) async {
	print("###" + args[0]); // prints only "###iPhone"
}
```

`args[1]` contains "Xs" and so on. So passing arguments with quotes does not work. Interestingly though, having those quotes embedded in the script call works:

```yaml
scripts:
  screenshots: fvm dart flutter_screenshots.dart "iPhone Xs Max"
```

This works just fine. The argument parser seems to not account for spaces in quotes.

**Comment by gonuit** [comment]
Hi @martin-braun 👋🏻,
What version of rps and dart are you using? I cannot reproduce this on the latest version of the package.

**Comment by martin-braun** [comment]
@gonuit 0.8.1, I haven't touched the project since then. If you cannot reproduce, maybe something in between messes up my args, but it's no issue for any other environment / scenario.

I'm launching my scripts from a tmux session, could this be related?

**Comment by gonuit** [comment]
Hi @martin-braun,
I managed to find the bug. The fix is available in version 0.8.2 and you can test it :)
